### PR TITLE
Feat: merge test js

### PR DESCRIPTION
**Pull request by Sotatek-MinhPham2**
Feat: merge test js

### CODE REVIEW

File: api/code-samples/javascript/klay/block/getCommittee.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const OpenSdk = require("opensdk-javascript");
+
+(() => {
+    const sdk = new OpenSdk(new Caver.ApiClient("https://api.baobab.klaytn.net:8651"));
+
```
Comment (Sotatek-HauTran3) [inline api/code-samples/javascript/klay/block/getCommittee.js:5]: Do you miss something here ?

File: sdk/client/javascript/openapi-test/test/klay/account/KlayEncodeAccountKeyApi.test.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('Klay account encode account key API', () => {
             expect(error).toBeNull();
             expect(data.jsonrpc).toBe("2.0");
             expect(data.result).toBeDefined()
-            console.log('API called successfully. Returned data: ' + JSON.stringify(data));
+            // console.log('API called successfully. Returned data: ' + JSON.stringify(data));
```
Comment (Sotatek-HauTran3) [inline sdk/client/javascript/openapi-test/test/klay/account/KlayEncodeAccountKeyApi.test.js:14]: remove unuse line

File: sdk/client/javascript/openapi-test/test/klay/account/KlayGetAccountApi.test.js [hunk 3]
```diff
@@ -3,15 +3,15 @@ const { expect } = require("@jest/globals");
...
     test('should return account.', (done) => {
 
-        let callbackOne = async function (error, data, response) {
-            await expect(error).toBeNull();
-            await expect(data.jsonrpc).toBe("2.0");
+        let callbackOne = function (error, data, response) {
+            expect(error).toBeNull();
+            expect(data.jsonrpc).toBe("2.0");
             expect(data.result).toBeDefined()
             //console.log('API called successfully. Returned data: ' + JSON.stringify(data));
```
Comment (Sotatek-HauTran3) [inline sdk/client/javascript/openapi-test/test/klay/account/KlayGetAccountApi.test.js:13]: remove this line, do not comment

File: sdk/client/javascript/openapi-test/test/klay/block/KlayBlockNumberApi.test.js [hunk 4]
```diff
@@ -7,7 +7,8 @@ describe('Klay block number API', () => {
     test('should return block number', (done) => {
 
         let callbackOne = function (error, data, response) {
-
+            if (error)
```
Comment (Sotatek-HauTran3) [inline sdk/client/javascript/openapi-test/test/klay/block/KlayBlockNumberApi.test.js:10]: it's seem this ut alway pass, right?

File: sdk/client/javascript/openapi-test/test/klay/block/KlayGetBlockByHashApi.test.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+const { expect } = require("@jest/globals");
+
+const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
+
+describe('Klay getBlockByHash API', () => {
+    test('should return block.', (done) => {
+
+        let callbackOne = function (error, data, response) {
+            console.log(data,error);
+            // expect(error).toBeNull();
```
Comment (Sotatek-HauTran3) [inline sdk/client/javascript/openapi-test/test/klay/block/KlayGetBlockByHashApi.test.js:11]: I think it should be uncommented here

File: api/code-samples/javascript/klay/block/getCommittee.js (hunk 1 above)
Comment (Sotatek-TanHoang) [reply api/code-samples/javascript/klay/block/getCommittee.js:5]: resolved

File: sdk/client/javascript/openapi-test/test/klay/account/KlayEncodeAccountKeyApi.test.js (hunk 2 above)
Comment (Sotatek-TanHoang) [reply sdk/client/javascript/openapi-test/test/klay/account/KlayEncodeAccountKeyApi.test.js]: resolved

File: sdk/client/javascript/openapi-test/test/klay/account/KlayGetAccountApi.test.js (hunk 3 above)
Comment (Sotatek-TanHoang) [reply sdk/client/javascript/openapi-test/test/klay/account/KlayGetAccountApi.test.js]: resolved

File: sdk/client/javascript/openapi-test/test/klay/block/KlayBlockNumberApi.test.js (hunk 4 above)
Comment (Sotatek-TanHoang) [reply sdk/client/javascript/openapi-test/test/klay/block/KlayBlockNumberApi.test.js]: resolved

**Comment by Sotatek-HauTran3** [comment]
LGTM

**Pull request merged**